src/main/java/uk/gov/hmcts/darts/event/service/handler/DarStopHandler.java

```diff
@@ -30,8 +31,8 @@ public DarStopHandler(RetrieveCoreObjectService retrieveCoreObjectService,
     @Override
     @Transactional
     public void handle(DartsEvent dartsEvent, EventHandlerEntity eventHandler) {
-        createHearingAndSaveEvent(dartsEvent, eventHandler); // saveEvent
-        var notifyEvent = new DarNotifyApplicationEvent(this, dartsEvent, STOP_RECORDING);
+        CreatedHearingAndEvent hearingAndSaveEvent = createHearingAndSaveEvent(dartsEvent, eventHandler);// saveEvent
+        var notifyEvent = new DarNotifyApplicationEvent(this, dartsEvent, STOP_RECORDING, hearingAndSaveEvent.getHearingEntity().getCourtroom().getId());
```

Comment:
minor: 
we could consider adding a helper method `getCourtRoomId()` on `CreatedHearingAndEvent` to avoid this long chained series of method calls.

Reply:
done